mantidproject/mantid
# Move Dead time to a Corrections tab in Muon Analysis

**Description of work.**
This PR moves the dead time corrections from the home tab of the Muon Analysis GUI to a new tab called the Corrections tab. Background corrections will be added to this tab in a future PR.

**To test:**
Open `Muon Analysis`
Load 84447
Go to the Corrections tab. You can see it has automatically calculated dead time `From file`.
Switch to `From workspace`. It will default to None - it should do calculation and then say 'No dead time corrections'
Choose a workspace and make sure it applies the corrections.
Switch to `From other file`. 
Try load the first data below. It should not crash, and should show an error popup.
Try load the second data below. It should select the loaded dead time workspace and calculate the dead time.

To test the `Runs` selector, load runs `500,90000` which have different dead times.
It should select the `From file` option automatically.
Select run 500 - the dead time should be zero.
Select run 90000 - the dead time should be non-zero.

Load 84447-9 and select co-add mode
The `Runs` selector should show `84447-84449` (because the corrections are performed after the summation)

Repeat for `Frequency Domain Analysis`

Fixes #31814

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
